prismic-model/src/parts/body.ts

```diff
@@ -15,12 +15,12 @@ import booleanDeprecated from './boolean-deprecated';
 type SliceProps = {
   nonRepeat?: Record<string, unknown>;
   repeat?: Record<string, unknown>;
+  description?: string;
 };
 
 export function slice(
   label: string,
-  { nonRepeat, repeat }: SliceProps,
-  description?: string
+  { nonRepeat, repeat, description }: SliceProps
```

Comment:
should we name label as well, does it look strange sitting separately?

Reply:
I guess since label is required (and would require a lot of changes elsewhere in the prismic-model directory) I'd be more tempted to keep it as is?

Reply:
yeah the other changes are the annoying bit, but I get your point about required or not... just feels strange when it gets displayed with description and they're at either ends of the function declaration.

Reply:
I think I did?

Reply:
oh yeah, ignore me!